stat.proto

```diff
@@ -50,6 +50,8 @@ message StatRequest {
     optional int32 end_point_id = 19;
     optional string end_point_name = 20;
     optional StatInfoResponse info_response = 21;
+    optional int32 token_id = 22;
+    optional string token = 23;
```

Comment:
`user_id`, `user_name`, `token_id`, `token` ... Seems a little bit too much.
What's the use of all this? Can't we just keep `token` (and clean the fat stat db on the move) ?